app/views/external_users/claims/fixed_fees/advocates/_fixed_fee_fields.html.haml

```diff
@@ -1,67 +1,61 @@
-          input_type: 'currency',
-          errors: @error_presenter,
-          input_readonly: f.object.price_calculated?,
-          value: number_with_precision(f.object.rate, precision: 2)
+        width: 3
+
+      = f.govuk_number_field :rate,
+        class:'rate fee-rate js-fee-calculator-rate',
+        form_group: {classes: 'js-unit-price-effectee calculated-unit-fee'},
+        label: { text: t('.rate_html', context: f.object.description) },
+        prefix_text: '£',
+        readonly: f.object.price_calculated?,
+        value: number_with_precision(f.object.rate, precision: 2),
+        width: 5
+
+      = f.govuk_text_field :price_calculated,
+        form_group: { classes: 'js-fee-calculator-success' },
+        label: { hidden: true },
+        type: 'hidden',
+        value: f.object.price_calculated?
 
-    .js-fee-calculator-success
-      = f.hidden_field :price_calculated, value: f.object.price_calculated?
-
-    .fee-calc-help-wrapper.form-group.hidden
       = govuk_detail t('.help_how_we_calculate_rate_title') do
         = t('.help_how_we_calculate_rate_body')
 
-    .dates-wrapper.form-group
-      .cocoon-insert-wrapper
-        - f.object.dates_attended.build unless f.object.dates_attended.any?
-        = f.fields_for :dates_attended do |date_attended|
-          = render 'date_attended_fields', f: date_attended, submodel_count: date_attended.index+1, parent_model_prefix: "fixed_fee_#{@fixed_fee_count}"
+      .dates-wrapper
+        .cocoon-insert-wrapper
+          - f.object.dates_attended.build unless f.object.dates_attended.any?
+          = f.fields_for :dates_attended do |date_attended|
+            = render partial: 'date_attended_fields', locals: { f: date_attended, submodel_count: date_attended.index+1, parent_model_prefix: "fixed_fee_#{@fixed_fee_count}" }
+
+        = link_to_add_association t('.add_date_attended'), f, :dates_attended, partial: 'date_attended_fields', data: { 'association-insertion-method' => 'append', 'association-insertion-node' => '.cocoon-insert-wrapper', 'association-insertion-traversal' => 'prev'}, class: 'govuk-link'
 
-      = link_to_add_association t('.add_date_attended'), f, :dates_attended, partial: 'date_attended_fields', data: { 'association-insertion-method' => 'append', 'association-insertion-node' => '.cocoon-insert-wrapper', 'association-insertion-traversal' => 'prev'}
+      = govuk_summary_list do
+        = govuk_summary_list_row do
+          = govuk_summary_list_key do
+            = t('.net_amount')
 
-    = govuk_summary_list do
-      = govuk_summary_list_row do
-        = govuk_summary_list_key do
-          = t('.net_amount')
+          = govuk_summary_list_value(class: 'fee-net-amount total') do
+            = fee.amount || number_to_currency(0)
 
-        = govuk_summary_list_value(class: 'fee-net-amount total') do
-          = fee.amount || number_to_currency(0)
+  = f.hidden_field :_destroy, class: 'destroy', value: f.object.blank?
```

Comment:
Nit: are we migrating these `hidden_field` helpers to the relevant design system helper?